01-Fundamentals-part-2/assignments2.js

```diff
@@ -109,4 +109,34 @@ if (mark.calcBmi() < john.calcBmi()) {
       john.fullName
     }'s (${john.bmi.toFixed(2)}))`
   );
+}*/
+console.log("Fundamentals part 2. Coding challenge #4");
+
+const bills = [22, 295, 176, 440, 37, 105, 10, 1100, 86, 52];
+const tips = [];
+const totals = [];
+let calcTip = (billValue) => {
```

Comment:
here, and for functions in general, better to use `const`, since you will unlikely reassign it.